docs/physics/new_plasma.rst

```diff
@@ -39,3 +39,10 @@ If the necessary Python modules (`PyGraphviz <https://pygraphviz.github.io/>`_ a
 Updating the Plasma
 -------------------
 During each iteration of the main code, TARDIS updates the plasma using the `update_radiationfield` function. This requires, at minimum, new values for `t_rad` (the radiation temperature), `w` (the dilution factor) and `j_blues` (the intensity in the blue part of each line).
+
+
```

Comment:
Here add a title for plasma calculations and then TOC within it. This will bring it on same level as other sections above in hierarchy shown in sidebar.

```
Overview of Plasma Calculations
--------------------------------------------
<copy the toctree within plasma/index.rst here (without title ofc since we have it in previous line)>
```
Then I guess you can also delete plasma/index.rst since it no longer be of any use.